src/components/Header/Header.jsx

```diff
@@ -71,43 +77,49 @@ export function Header(props) {
   // Reports
   const canGetReports = props.hasPermission('getReports');
   const canGetWeeklySummaries = props.hasPermission('getWeeklySummaries');
+  const canGetWeeklyVolunteerSummary = props.hasPermission('getWeeklySummaries');
```

Comment:
The permission "getWeeklySummaries" is used for both `canGetWeeklySummaries` and `canGetWeeklyVolunteerSummary`. Do we need a separate constant for the same permission?

Reply:
we still need to configure other APIs to send permission specifically for Total Org Summary. Currently, it is just using permission for WeeklySummaries. Permission will be configured once we have a design ready.

Reply:
Thanks for the explanation and for sharing the future plans for this.